docs/about/release/linux_support.md

```diff
@@ -1,6 +1,6 @@
-# GPU Support and OS Compatibility (Linux)
+# GPU and OS support (Linux)
 
-(supported_distributions)=
+(linux_support)=
 
 ## Supported Linux Distributions
```

Comment:
This is in upper case. Is it intended?

Reply:
Nope, but I'll deal with text consistency in a different PR (after the structural stuff is finished).